cns/grpc/proto/server_grpc.pb.go

```diff
@@ -0,0 +1,152 @@
+// Code generated by protoc-gen-go-grpc. DO NOT EDIT.
```

Comment:
Same as above, let's remove this file from the `proto/` directory

Reply:
I git ignored this file as well